README.md

```diff
@@ -70,3 +70,7 @@ Tags, colors and clusters help to retain the overview.
 <a href="https://opencollective.com/utopia-project">
     <img width="300" src="https://opencollective.com/utopia-project/donate/button@2x.png?color=blue" />
 </a>
+
+---
+
+This project is tested with Browserstack.
```

Comment:
Thank you for thinking of that.
Maybewe could include their logo (e.g. as we did in the [Ocelot repo Readme](https://github.com/Ocelot-Social-Community/Ocelot-Social/blob/master/README.md)).